proto/utxorpc/v1alpha/sync/sync.proto

```diff
@@ -64,6 +64,7 @@ message ReadTipRequest {}
 // Response containing the current tip of the blockchain.
 message ReadTipResponse {
   BlockRef tip = 1; // The current tip of the blockchain.
+  uint64 timestamp = 2; // Block ms timestamp
```

Comment:
wasn't sure the best way to handle this case. There are two options:
1. I can include a `timestamp` field here directly
2. I can change this so that instead of returning `BlockRef`, it returns something like `BlockHeader` (but maybe `BlockHeader` may drag in too much data, and also see other discussion about tradeoffs for inclusion in `BlockHeader`)

Note: adding it to `BlockRef` directly isn't a great option, since `BlockRef` is used in other places where supporting a timestamp field is not ideal